apps/cookbook/src/app/showcase/list-load-on-demand-showcase/list-load-on-demand-showcase.component.html

```diff
@@ -1,4 +1,29 @@
 <div class="example">
+  <p>
+    The kirby list will emit at <code>loadMore</code> event, when the user has scrolled 80% of the
+    list. It is the responsibility of the consumer to add more items to the bound
+    <code>items</code>.
+  </p>
+  <p>
+    The event emitted is of type <code>KirbyLoadMoreEvent</code> which exposes a
+    <code>complete</code> callback function.
+  </p>
+  <p>
+    The <code>complete</code> callback function should be called when the loading of the additional
+    items are done.
+  </p>
+  <p>
+    If <code>complete</code> is called with the <code>disableLoadMore</code> argument set to
+    <code>true</code>, no further events will be emitted. Further more, if any is given, the
+    <code>noMoreItemsText</code> will be shown.
+  </p>
   <cookbook-list-load-on-demand-example></cookbook-list-load-on-demand-example>
   <cookbook-code-viewer [html]="exampleHtml"></cookbook-code-viewer>
+  <h2>Properties:</h2>
+  <cookbook-showcase-properties [properties]="properties"></cookbook-showcase-properties>
+  <h2>Events:</h2>
```

Comment:
Consider importing `@import '../showcase.shared';` in `apps/cookbook/src/app/examples/list-load-on-demand-example/list-load-on-demand-example.component.scss` to get default spacing between headings (e.g. see https://cookbook.kirby.design/home/showcase/modal and `apps/cookbook/src/app/showcase/modal-showcase/modal-showcase.component.scss` for an example)

Reply:
Resolved by: 6e9a684